test/unit/core/system.ts

```diff
@@ -41,13 +42,24 @@ suite('System', ({ expect, spy, stub }) => {
       expect(initServices).to.be.called;
       expect(initMixin).to.be.called;
       expect(registerTags).to.be.called;
+      expect(initUserMixins).to.be.called;
+      sinon.assert.callOrder(
+        initConfig,
+        initRiot,
+        initFlux,
+        createServices,
+        initServices,
+        initMixin,
+        registerTags,
+        initUserMixins
+      );
```

Comment:
We could potentially create an array of the functions in order, loop through it asserting that each one is called, then assert that `initConfig` and `createServices` are called with the proper arguments, then destructure the array in `callOrder`.

Reply:
I think it's fine